src/raven.js

```diff
@@ -1,12 +1,31 @@
-/*global XDomainRequest:false, __DEV__:false*/
+/*global XDomainRequest:false */
 
 var TraceKit = require('../vendor/TraceKit/tracekit');
 var stringify = require('../vendor/json-stringify-safe/stringify');
 var RavenConfigError = require('./configError');
-var utils = require('./utils');
 
-var isError = utils.isError,
-  isObject = utils.isObject;
+var utils = require('./utils');
```

Comment:
I didn't want to change all instances to `utils.something` in the codebase, but rather just pull them out like this to make this PR "easier" to digest.